src/header.css

```diff
@@ -19,6 +19,10 @@
     color: #555
 }
 
+.great-header .notification-indicator {
+    color: #4078c0;
```

Comment:
I think this should be #555 or a similar neutral gray to match the other icons. There is a separate blue icon if you have any notifications.

Reply:
Thanks for the feedback. I did wonder about that when I wrote it.

I have now styled it `#333` and blue on hover, just like the other links in the header and the + icon beside it.